app/views/candidates/schools/_results.html.erb

```diff
@@ -58,16 +54,23 @@
         <hr />
 
         <ul id="results">
-            <% @search.results.each_with_index do |result, index| %>
-            <li class="school-result" data-school-urn="<%= result.urn %>">
-                <strong class="name"><%= link_to result.name, candidates_school_path(result) %></strong>
-                <ul>
-                    <li><strong>Address:</strong> <%= format_school_address result %></li>
-                    <li><strong>Education phase:</strong> <%= format_school_phases result %></li>
-                    <li><strong>Fees:</strong> &pound;<%= result.fee %></li>
-                    <li><strong>School type:</strong> <%= result.school_type&.name %></li>
-                    <li><strong>Subjects:</strong> <%= format_school_subjects result %></li>
-                </ul>
+            <% @search.results.each_with_index do |school, index| %>
+            <li data-school-urn="<%= school.urn %>">
+              <div class="school-result">
+								<h3 class="govuk-heading-l">
+                  <%= school.name %>
+									<span class="govuk-caption-m">{{'1 mile' if item.Distance == 1 else item.Distance + ' miles'}} away</span>
```

Comment:
Looks like some left over nunjucks

Reply:
Yeh, thats there as a reminder :)

Need to discuss distances with Pete before I can fix that. Have realised I've got some mucked up indentation as well.

Realised I've replace the start page with the splash page instead of adding it as a revision so will put this back to draft for now.

Thanks

Reply:
Hmm, can't make it draft, will ping you when I'm ready for re-review

Reply:
When there is a location component to the search, there should be a distance returned with each result:

```ruby
Bookings::School.close_to(some_point, radius: 70).map(&:distance)
=> [99710.19683742,
 71377.83624106,
 37895.58175206,
```

The distance is in metres so that'll need converting but should be straightforward to display in the listing.